setup.cfg

```diff
@@ -16,21 +16,21 @@ classifiers =
     Framework :: napari
     Programming Language :: Python
     Programming Language :: Python :: 3
-    Programming Language :: Python :: 3.8
     Programming Language :: Python :: 3.9
     Programming Language :: Python :: 3.10
+    Programming Language :: Python :: 3.11
+    Programming Language :: Python :: 3.12
     Operating System :: OS Independent
     License :: OSI Approved :: BSD License
 
 
 [options]
 zip_safe = False
 packages = find:
-python_requires = >=3.8
+python_requires = >=3.9
 include_package_data = True
 install_requires =
-	imageio
-	imageio-ffmpeg
+	imageio[ffmpeg]
```

Comment:
This should be functionally equivalent, but fewer CI jobs failed when imageio & ffmpeg were installed as a single step.